grouper/fe/handlers.py

```diff
@@ -858,6 +888,172 @@ def get(self, group_id=None, name=None):
         )
 
 
+class GroupPermissionRequest(GrouperHandler):
+    def _get_forms(self, args_by_perm, get_request_data=False):
+        if get_request_data:
```

Comment:
This behaviour seems a bit magic. I'd prefer that this be refactored into a `@staticmethod` that always took the data as a parameter (or `None`)